lib/mapper.js

```diff
@@ -108,6 +113,53 @@ function formatCompany(company){
   return ret;
 }
 
+ /**
+ * Formats location data for use with intercom
+ *
+ * https://doc.intercom.io/api/#user-model
+ *
+ * @param {Object} location
+ * @return {Object}
+ * @api private
+ */
+
+function formatLocation(context){
+  var location = context.location || {};
+  var codes = getCountryCodes(location.country);
```

Comment:
`codes` doesn't tell me much about what this variable stores. I'd break it into two separate variables—countryCode and continentCode